### PR TITLE
ETS-1809 Typography

### PR DESCRIPTION
**Pull request by BirgitPohl**
## 0.35.0
- Updates the typography
- Additionally specifically defines typography for teasers and header elements
- Renames usable to undefined: Making it possible to explicitly define elements, that are not defined inside design-guide
- Some clean-up refactoring

### CODE REVIEW

**Review by xavero-evandro — DISMISSED** [review]

File: source/_patterns/04-pages/00-typography.mustache [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <main role="main" class="container">
   <h2>Headlines</h2>
 
-  {{> atoms-h1 (content: 'This is a simple <H1> headline') }}
-  {{> atoms-h2 (content: 'This is a simple <H2> headline') }}
-  {{> atoms-h3 (content: 'This is a simple <H3> headline') }}
-  {{> atoms-h4 (content: 'This is a simple <H4> headline') }}
-  {{> atoms-h5 (content: 'This is a simple <H5> headline') }}
+  {{> atoms-h1(content: "This is a simple <H1> headline") }}
```
Comment (filipe-esposito-fti) [inline source/_patterns/04-pages/00-typography.mustache]: Why was this changed to double quotes? All the following lines were kept with single quotes.

Comment (BirgitPohl) [reply source/_patterns/04-pages/00-typography.mustache]: Good point. :)

**Review by filipe-esposito-fti — DISMISSED** [review]
